src/ui/public/index_patterns/_get.js

```diff
@@ -34,13 +34,19 @@ export function IndexPatternsGetProvider(Private) {
     });
   };
 
-  return (field) => {
+  function retFunction(field) {
```

Comment:
Perhaps instead of using a function definition, you can use a function expression to avoid unnecessary hoisting?

```js
const retFunction = field => {
```